/*Write an application named BadSubscriptCaught in which you declare an array of 10 first names.
Write a try block in which you prompt the user for an integer and display the name in the requested position.
Create a catch block that catches the potential ArrayIndexOutOfBoundsException thrown when the user
enters a number that is out of range. The catch block should also display an error message.
 */
/******************************************************************************
 Programmer: Nicholas Foley
 Date: 02/20/2023
 Lab 12
 Instructor: Dr. Rafael Azuaje
 College: Northeast Lakeview
 *******************************************************************************/
public class BadSubscriptCaught {
    public static void main(String[] args) {

        String[] array = { "Albert", "Billy", "Charles", "Daniel", "Elizabeth",
                "Francine", "George", "Henry", "Igor", "Jacen"};
    }

}